src/transformers/models/oneformer/processing_oneformer.py

```diff
@@ -76,13 +100,46 @@ def _preprocess_text(self, text_list=None, max_length=77):
         token_inputs = torch.cat(token_inputs, dim=0)
         return token_inputs
 
-    def __call__(self, images=None, task_inputs=None, segmentation_maps=None, **kwargs):
```

Comment:
These should be added to the optional_call_args attribute (see [udop processor](https://github.com/huggingface/transformers/blob/6c3f168b36882f0beebaa9121eafa1928ba29633/src/transformers/models/udop/processing_udop.py#L81))